src/Tribe/Integrations/WPML/WPML.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+
+/**
+ * Class Tribe__Events__Integrations__WPML__WPML
+ *
+ * Handles anything relating to The Events Calendar and WPML integration
+ *
+ * This class is meant to be an entry point hooking specialized classes and not
+ * a logic hub per se.
+ */
+class Tribe__Events__Integrations__WPML__WPML {
+
+	/**
+	 * @var Tribe__Events__Integrations__WPML__WPML
+	 */
+	protected static $instance;
+
+	/**
+	 * The class singleton constructor.
+	 *
+	 * @return Tribe__Events__Integrations__WPML__WPML
+	 */
+	public static function instance() {
```

Comment:
As with `Tribe__Events__Integrations__Manager` - if we want to enforce there being only one object of this type we should probably define a protected/private constructor, else it's implicitly public and instances can be created willy-nilly.